LiveSplit.AutoSplitters.xml

```diff
@@ -8069,10 +8069,10 @@
             <Game>Quantum Conundrum</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/blairmadison11/QuantumConundrumAutosplitter/master/QuantumConundrum.asl</URL>
+            <URL>https://raw.githubusercontent.com/Tesfallout/QuantumConundrumAutosplitter/main/QuantumConundrum.asl</URL>
         </URLs>
         <Type>Script</Type>
-        <Description>Autosplitting and Load Removal is available. (by blairmadison11)</Description>
+        <Description>Complete rewrite to improve Autostart, Autosplitting, and Load Removal. (by Tesfallout)</Description>
```

Comment:
I would recommend not wording it this way. The users of this auto splitter do not care that it was rewritten, just about what features it possesses.

Reply:
Updated description.  Blairmadison11's script was made quite a few years ago.  If they are not reachable, will a screenshot of a community vote suffice?

Reply:
So you have in fact *not* received permission from the author to replace the existing auto splitter, like you checked in the PR?

Reply:
Sorry, this is my first time doing this and I did not see anything addressing what the procedure is for when you have no idea how to contact someone who has not been in the community for years. Will a screenshot of a community poll be sufficient?

Reply:
A response from the author would be preferred.

Reply:
If you've tried to reach the original author and they're not responding, and the community moderators are OK with replacing their auto splitter, then it's fine to merge this

Reply:
Alright, I've put a poll up and I'll see if anyone knows how to contact blairmadison11.  It looks like they haven't been around for 6 years, so only time will tell.

Reply:
They were active here on GitHub around half a year ago. I would give it maybe a day or two to potentially see the tag from my comment below (https://github.com/LiveSplit/LiveSplit.AutoSplitters/pull/1314#issuecomment-1529021440).

Reply:
Ok, We've got approval from the src mods and blair has messaging blocked on twitch and twitter.  Just let me know what you need from here on.

Reply:
Hi all, sorry for the delayed response! I'm totally fine with my autosplitter script being replaced as long as I am still given credit for doing the initial version. It seems like the new script builds on my work (using the memory addresses and values that I spent a lot of time reverse engineering), so it would be nice to be credited. Also, I'm just going to assume that the new script has been tested extensively. I seem to recall that those loading flags were needed to prevent splits during loads from menu, but if you can do it all with just game state, then that's fine. As long as it works, it's all good.